idris-commands.el

```diff
@@ -74,8 +74,9 @@
 
 (defun idris-ensure-process-and-repl-buffer ()
   "Ensure that an Idris process is running and the Idris REPL buffer exists."
-  (idris-repl-buffer)
-  (idris-run))
+  (idris-run)
+  (with-current-buffer (idris-repl-buffer)
+    (goto-char (point-max))))
```

Comment:
is this `(goto-char (point-max))` really intentional? do all callers of ensure-p-and-r-b need this?